core/src/main/java/org/radargun/stages/test/Stressor.java

```diff
@@ -160,7 +160,7 @@ private void runInternal() {
       if (useTransactions && txRemainingOperations <= 0) {
          try {
             ongoingTx = stage.transactional.getTransaction();
-            logic.transactionStarted();
+            ongoingTx.wrap(invocation.getTxResource());
```

Comment:
This doesn't look right; you're not using the return value. You're supposed to call the operation methods on the return value (wrapper), not on the original cache. If you start a transaction but run the method (e.g. put()) on the original cache, it won't run transactionally in Hazelcast.

Reply:
Thanks for catching this @rvansa 
I am closing this PR